src/pages/MenuPage.js

```diff
@@ -26,22 +26,22 @@ const SECTIONS = {
   customer: {
     icon: require('../images/menu_people.png'),
     buttons: [
-      { title: navStrings.customer_invoices, page: 'customerInvoices' },
-      { title: navStrings.customer_requisitions, page: 'customerRequisitions' },
+      { title: 'customer_invoices', page: 'customerInvoices' },
+      { title: 'customer_requisitions', page: 'customerRequisitions' },
     ],
   },
   supplier: {
     icon: require('../images/menu_truck.png'),
     buttons: [
-      { title: navStrings.supplier_invoices, page: 'supplierInvoices' },
-      { title: navStrings.supplier_requisitions, page: 'supplierRequisitions' },
+      { title: 'supplier_invoices', page: 'supplierInvoices' },
+      { title: 'supplier_requisitions', page: 'supplierRequisitions' },
     ],
   },
   stock: {
     icon: require('../images/menu_pc_clipboard.png'),
     buttons: [
-      { title: navStrings.current_stock, page: 'stock' },
-      { title: navStrings.stocktakes, page: 'stocktakes' },
+      { title: 'current_stock', page: 'stock' },
```

Comment:
i know this i of my doing, but if we make this a method (like you've done for IC customisation, can revert back to navStrings and don't need to do the `navStrings[title]` shenanigans in renderButton ) and

Reply:
d94ade22813c4579ce9acbc7f1245228bc612005